_posts/2017-05-30-database-encryption.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "Database encryption"
+date: 2017-05-30 11:03
+comments: true
+author: ashkan
+categories: [Database, Encryption, Security]
+---
+
+Recently after examining the data stored in one of our systems, we noticed that while originally this system wasn't designed to include sensitive data, over the time it ended up including some sensitive information and we need to encrypt the data.
+
+While our systems were safe by other means, because of the issue above we decided to encrypt the database behind it to cover the cases when people end up accessing our database directly. This means the data needs to be encrypted before getting persisted in the database and also decrypted after retrieval so from the consumer perspective nothing has changed.
```

Comment:
Yes, I like this. It would also answer somewhat to my questions about who we’re guarding against.